aws-auth-cognito/src/test/resources/feature-test/testsuites/resetPassword/AuthResetPasswordResult_object_is_returned_when_reset_password_succeeds.json

```diff
@@ -0,0 +1,77 @@
+{
+    "description": "AuthResetPasswordResult object is returned when reset password succeeds",
+    "preConditions": {
+        "amplify-configuration": "authconfiguration.json",
+        "state": "SignedIn_SessionEstablished.json",
+        "mockedResponses": [
+            {
+                "type": "cognito",
+                "apiName": "forgotPassword",
+                "responseType": "success",
+                "response": {
+                    "codeDeliveryDetails": {
+                        "destination": "dummy destination",
+                        "deliveryMedium": "EMAIL",
+                        "attributeName": "dummy attribute"
+                    }
+                }
+            }
+        ]
+    },
+    "api": {
+        "name": "resetPassword",
+        "params": {
+            "username": "someUsername"
+        },
+        "options": {
+        }
+    },
+    "validations": [
+        {
+            "type": "cognito",
+            "apiName": "forgotPassword",
+            "request": {
+                "username": "someUsername",
+                "clientId": "testAppClientId",
+                "clientMetadata": {
+                }
+            }
+        },
+        {
+            "type": "amplify",
+            "apiName": "resetPassword",
+            "responseType": "success",
+            "response": {
+                "isPasswordReset": false,
+                "nextStep": {
+                    "resetPasswordStep": "CONFIRM_RESET_PASSWORD_WITH_CODE",
+                    "additionalInfo": {
+                    },
+                    "codeDeliveryDetails": {
+                        "destination": "dummy destination",
+                        "deliveryMedium": "EMAIL",
+                        "attributeName": "dummy attribute"
+                    }
+                }
+            }
+        },
+        {
+            "type": "amplify",
+            "apiName": "resetPassword",
+            "responseType": "success",
+            "response": {
+                "isPasswordReset": false,
+                "nextStep": {
+                    "resetPasswordStep": "CONFIRM_RESET_PASSWORD_WITH_CODE",
+                    "additionalInfo": {
+                    },
+                    "codeDeliveryDetails": {
+                        "destination": "dummy destination",
+                        "deliveryMedium": "EMAIL",
+                        "attributeName": "dummy attribute"
+                    }
+                }
+            }
+        }
```

Comment:
Duplicate validation... It is the same as the one above.

Reply:
nice catch! these files would be eventually moved out of this repo eventually. I will fix them in follow up PR